public/manifest.json

```
@@ -8,7 +8,8 @@
     "activeTab",
     "storage",
     "declarativeContent",
-    "https://github.com/"
+    "https://github.com/",
+    "https://trello.com/"
```

```suggestion
    "https://github.com/"
```
`https://trello.com/` はなくても動くと思ったのですが、いかがでしょうか。（少なくともこちらの手元では動作しました。）

`https://github.com/` のみ指定されている理由は以下に記載されております。
https://github.com/twin-te/manaba-report-integration/pull/28